CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [0.0.next] - 2017-06-02
```

Comment:
This should just be be `[Unreleased]` we will bump the version on release (I usually do within 24 hours max) of when I have merged something.